composer-lock-diff

```diff
@@ -47,22 +47,23 @@ function diff($key, $from, $to, $base_path) {
     $data = load($from, $base_path);
 
     foreach($data->$key as $pkg) {
-        $pkgs[$pkg->name] = array(version($pkg), 'REMOVED', '');
+        $pkgs[$pkg->name] = array($pkg->name, version($pkg), 'REMOVED', '', sourceUrl($pkg));
```

Comment:
Changing the offsets here has some side-effects. The one I noticed is that `makeCompareUrl()` relies on them and now the range isn't correct. Any easy fix though, just move the offsets here, https://github.com/davidrjonas/composer-lock-diff/blob/master/composer-lock-diff#L213

Reply:
Fixed the side effect of changing the top element of the array in json.